XIVSlothCombo/Combos/PvE/DRG.cs

```diff
@@ -200,123 +203,142 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
+                                                if (ActionReady(DragonfireDive))
                                                     return DragonfireDive;
-                                                if (LevelChecked(SpineshatterDive) && GetRemainingCharges(SpineshatterDive) > 0)
+
+                                                if (ActionReady(SpineshatterDive) && GetRemainingCharges(SpineshatterDive) > 0)
                                                     return SpineshatterDive;
                                             }
                                         }
                                     }
                                 }
-
-                                // healing - please move if not appropriate this high priority
-                                if (IsEnabled(CustomComboPreset.DRG_ST_ComboHeals))
-                                {
-                                    if (PlayerHealthPercentageHp() <= PluginConfiguration.GetCustomIntValue(Config.DRG_STSecondWindThreshold) && LevelChecked(All.SecondWind) && IsOffCooldown(All.SecondWind))
-                                        return All.SecondWind;
-                                    if (PlayerHealthPercentageHp() <= PluginConfiguration.GetCustomIntValue(Config.DRG_STBloodbathThreshold) && LevelChecked(All.Bloodbath) && IsOffCooldown(All.Bloodbath))
-                                        return All.Bloodbath;
-                                }
                             }
                         }
 
+                        // healing - please move if not appropriate this high priority 
+                        if (IsEnabled(CustomComboPreset.DRG_ST_ComboHeals))
+                        {
+                            if (PlayerHealthPercentageHp() <= PluginConfiguration.GetCustomIntValue(Config.DRG_STSecondWindThreshold)
+                                && ActionReady(All.SecondWind))
+                                return All.SecondWind;
+
+                            if (PlayerHealthPercentageHp() <= PluginConfiguration.GetCustomIntValue(Config.DRG_STBloodbathThreshold) &&
+                                ActionReady(All.Bloodbath))
+                                return All.Bloodbath;
+                        }
+
                         //1-2-3 Combo
                         if (HasEffect(Buffs.SharperFangAndClaw))
                             return FangAndClaw;
+
                         if (HasEffect(Buffs.EnhancedWheelingThrust))
                             return WheelingThrust;
+
                         if (comboTime > 0)
                         {
-                            if (ChaosDoTDebuff is null || ChaosDoTDebuff.RemainingTime < 6 || GetBuffRemainingTime(Buffs.PowerSurge) < 10)
+                            if (ChaosDoTDebuff is null || ChaosDoTDebuff.RemainingTime < 6 ||
+                                GetBuffRemainingTime(Buffs.PowerSurge) < 10)
```

Comment:
There is a error (issue #1203 ) -> GetBuffRemainingTime(Buffs.PowerSurge) < 10 need to be in side exclusive for the action. In low level the var ChaosDoTDebuff is allways null -> check for remainingtime skipped.